src/pages/jobs.js

```diff
@@ -0,0 +1,160 @@
+import { useState } from 'react'
+import Form from '../components/Jobs/Form/Form'
+import Card from '../components/Jobs/Card/Card'
+import Loader from '../components/Jobs/Loader/Loader'
+import Video from '../components/Jobs/Video/Video'
+import Pagination from '../components/Pagination/Pagination'
+import PageHeader from '../components/PageHeader'
+import { StaticImage } from 'gatsby-plugin-image'
+import SEO from '../components/SEO'
+
+function Jobs() {
+  const [jobData, setJobData] = useState(false)
```

Comment:
Why is the initial value a boolean when the data is a string or object? Try `useState('')`

Reply:
jobData is now an empty object.